File: src/main/java/no/nav/domain/pensjon/regler/grunnlag/ForstegangstjenestePeriode.java
package no.nav.domain.pensjon.regler.grunnlag;

import no.nav.domain.pensjon.regler.kode.ForstegangstjenesteperiodeTypeCti;

import java.io.Serializable;
import java.util.Date;

public class ForstegangstjenestePeriode implements Serializable {

    private Date fomDato;
    private Date tomDato;
    private ForstegangstjenesteperiodeTypeCti periodeType;


}

File: src/main/kotlin/no/nav/pensjon/regler/domain/grunnlag/ForstegangstjenestePeriode.kt
package no.nav.pensjon.regler.domain.grunnlag

import no.nav.pensjon.regler.domain.enum.ForstegangstjenestetypeEnum
import java.io.Serializable
import java.util.*
import java.time.LocalDate

class ForstegangstjenestePeriode : Serializable {
    var fomDatoLd: LocalDate? = null
    var tomDatoLd: LocalDate? = null
    var periodeTypeEnum: ForstegangstjenestetypeEnum? = null
}